### DISCUSSION TITLE
Build a website from an HTML website

### DISCUSSION THREAD

**Discussion by rinka88** [templaza/astroid-framework#796] [General]
@sonvnn  I want to assemble a website from ready-made HTML landing sites and transfer it to Astroid using widgets. All styles and JS files were included from the template settings. They are visible on the page. And when applying styles to a section, for some reason it doesn’t pick up. All styles are compiled into a compressed file and taken from there, but I would like it to be taken from each file separately. Compression is disabled in the settings, but it still compresses. Joomla and Astroid latest versions. Astroid child template.

**Comment by Chacapamac** [comment]
Do you see the CSS files loaded in the page/menu you need them?

**Comment by rinka88** [comment]
Naturally, the first thing I checked was this.
All styles and js files are included.
but for some reason I can’t see the auxiliary class styles from the new design.
And the compiled style file is included, although I do not have compilation enabled

**Comment by Chacapamac** [comment]
Ok I see.

> And when applying styles to a section, for some reason it doesn’t pick up.

Maybe an interface error, do you see any console or server error when changing/applying new style?
